airflow/stats.py

```diff
@@ -30,6 +31,245 @@
 log = logging.getLogger(__name__)
 
 
+class MetricType(str, enum.Enum):
+    """Metrics types."""
+
+    COUNTER = "COUNTER"
+    GAUGE = "GAUGE"
+    TIMER = "TIMER"
+
+
+class Metric(NamedTuple):
+    """StatsD metrics definition"""
+
+    metric_type: str
+    key: str
+    description: str
+
+
+METRICS_LIST: List[Metric] = [
```

Comment:
I'm not sure the purpose of this List, nor the Metric class -- it appears this is never used for emitting metrics at runtime, so I'm not sure why we need this?

What was your thinking here please?